### PR TITLE
Feature/white stork kml

### PR DESCRIPTION
**Pull request by onyxvd**
Show local projects kml

### CODE REVIEW

File: app/src/main/java/org/bspb/smartbirds/pro/ui/map/OsmMapProvider.java [hunk 1]
```diff
@@ -184,6 +186,11 @@ public void setShowZoneBackground(boolean showBackground) {
         drawZones();
     }
 
+    @Override
+    public void setShowLocalProjects(boolean showKml) {
+
+    }
```
Comment (groupsky) [inline app/src/main/java/org/bspb/smartbirds/pro/ui/map/OsmMapProvider.java:192]: not implemented?

File: app/src/main/java/org/bspb/smartbirds/pro/ui/map/GoogleMapProvider.java [hunk 2]
```diff
@@ -198,6 +213,28 @@ public void drawPath(List<LatLng> points) {
 
     }
 
+    private void drawLocalProjects(boolean showKml) {
+        if (fragment == null || fragment.getContext() == null) {
+            return;
+        }
+
+        if (showKml) {
```
Comment (groupsky) [inline app/src/main/java/org/bspb/smartbirds/pro/ui/map/GoogleMapProvider.java:221]: calling twice `drawLocalProjects(true)` will duplicate the markers.

File: app/src/main/java/org/bspb/smartbirds/pro/ui/map/OsmMapProvider.java (hunk 1 above)
Comment (onyxvd) [reply app/src/main/java/org/bspb/smartbirds/pro/ui/map/OsmMapProvider.java]: Will implement after finish with OSM dependency upgrade

File: app/src/main/java/org/bspb/smartbirds/pro/ui/map/GoogleMapProvider.java [hunk 3]
```diff
@@ -426,7 +468,17 @@ private void displayGeometry(KmlGeometry geometry, Style style) {
...
+            }
+        } else if (geometry instanceof KmlPoint) {
+            KmlPoint kmlPoint = (KmlPoint) geometry;
+            ArrayList<GeoPoint> geopoints = kmlPoint.mCoordinates;
+            for (GeoPoint point : geopoints) {
+                MarkerOptions markerOptions = new MarkerOptions()
+                        .position(new LatLng(point.getLatitude(), point.getLongitude()))
+                        .icon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_BLUE))
+                        .title(name);
+                mMap.addMarker(markerOptions);
```
Comment (groupsky) [inline app/src/main/java/org/bspb/smartbirds/pro/ui/map/GoogleMapProvider.java:481]: Is this needed for the stork local project? It seems to me storks are handled separately inside drawLocalProjects and this makes me think would it conflict with each other.

Comment (onyxvd) [reply app/src/main/java/org/bspb/smartbirds/pro/ui/map/GoogleMapProvider.java:481]: This allows drawing points from area.kml file.
I won't conflict with local projects.

**Pull request merged**